<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="26228" started-at="2019-12-21T02:32:02Z" finished-at="2019-12-21T02:32:28Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="26228" started-at="2019-12-21T02:32:02Z" finished-at="2019-12-21T02:32:28Z">
      <class name="com.cm.qa.testcases.UsernamePageTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="setUp" is-config="true" duration-ms="19632" started-at="2019-12-21T02:32:02Z" finished-at="2019-12-21T02:32:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="usernameScreenCMLogoImageTest()[pri:1, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="usernameScreenCMLogoImageTest" duration-ms="117" started-at="2019-12-21T02:32:22Z" finished-at="2019-12-21T02:32:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- usernameScreenCMLogoImageTest -->
        <test-method status="FAIL" signature="blankUsernameTest()[pri:2, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="blankUsernameTest" duration-ms="973" started-at="2019-12-21T02:32:22Z" finished-at="2019-12-21T02:32:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Must input username or email!
Error] but found [Must Enter User Name
User name not entered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Must input username or email!
Error] but found [Must Enter User Name
User name not entered]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.cm.qa.testcases.UsernamePageTests.blankUsernameTest(UsernamePageTests.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankUsernameTest -->
        <test-method status="FAIL" signature="verifyFooterText()[pri:3, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="verifyFooterText" duration-ms="99" started-at="2019-12-21T02:32:23Z" finished-at="2019-12-21T02:32:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Copyright © 2019 Exela Contract Management.
All Rights Reserved.
v0.0.8] but found [Copyright © 2019 Exela Contract Management.
All Rights Reserved.
v0.0.9]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Copyright © 2019 Exela Contract Management.
All Rights Reserved.
v0.0.8] but found [Copyright © 2019 Exela Contract Management.
All Rights Reserved.
v0.0.9]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.cm.qa.testcases.UsernamePageTests.verifyFooterText(UsernamePageTests.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFooterText -->
        <test-method status="PASS" signature="validUsernameTest()[pri:4, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="validUsernameTest" duration-ms="577" started-at="2019-12-21T02:32:23Z" finished-at="2019-12-21T02:32:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validUsernameTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.cm.qa.testcases.UsernamePageTests@fb434]" name="tearDown" is-config="true" duration-ms="4750" started-at="2019-12-21T02:32:24Z" finished-at="2019-12-21T02:32:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.cm.qa.testcases.UsernamePageTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
